module/runtime.go

```diff
@@ -0,0 +1,42 @@
+package module
+
+import (
+	"context"
+
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+)
+
+// APIModel is a glorified (resource.API, resource.Model) pair. Pass as many of these as you want
+// to ModularMain, to have this module support those models.
+type APIModel = resource.APIModel
```

Comment:
Can we find a better name for this? I  would rather not have `module.APIModel` and `resource.APIModel` because they will mean different things now. It might be okay, but I wonder why not just use `resource.APIModel` everywhere? A module does semantically mean a different thing - it's a little mini-server that houses models and their implementations outside the rdk.

Reply:
Yeah, you make a good point. I can remove it, but then you need to import two different packages (`resource` and `module`) to use this. Maybe there should be a standalone struct in here that happens to be a copy of `resource.APIModel`, and then you only need the one package. and that way, someone in the future could modify `resource.APIModel` in some extra way, and this code is unaffected.

Reply:
...and I shouldn't make a copy, because that makes the confusion even worse: now they really might be different! Perhaps the right approach really is to force module writers to import 2 separate packages into their main programs.